version.ts

```diff
@@ -0,0 +1 @@
+export const version = '1.0.0';
```

Comment:
Can't we use something like:
`export class AppUtil {
     getAppVersion = () => {
       return require( './package.json' ).version;
     }
}`